### DISCUSSION TITLE
Welcome to DKAN Discussions!

### DISCUSSION THREAD

**Discussion by dafeder** [GetDKAN/dkan#3813] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community, as we phase out our Slack and other channels to consolidate here. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that everyone here is doing their best to juggle priorities and building software is hard (but also exciting and rewarding).

As time goes on, the DKAN core team will use this space for things that are important to share but don't quite fit in issues or PRs, attempting to communicate a clearer picture of where DKAN is going and to crowdsource ideas as much as possible.

To help get us started, please comment below with an introduction of yourself, your project, and what you do or want to do within this community.

**Comment by alex-ahumada** [comment]
Hi! I'm Alex. I currently work at Cádiz City Council (Spain), we just released our [Open Data Portal ](https://datos.cadiz.es/) using DKAN.

**Comment by dafeder** [comment]
Hi! I'm Dan Feder, a software engineer at [CivicActions](https://github.com/CivicActions/). I'm one of the maintainers of DKAN and have been working on it since 2014. I spend most of my time working for public-sector clients, and at the moment am focused on https://openpaymentsdata.cms.gov/. I'm excited about what we've been able to accomplish with DKAN v2 and looking forward to making improvements that will make it easier for data publishers around the world to adopt and build on.
